README.md

```diff
@@ -1,83 +1,182 @@
+   For edits such as punctuation, formatting, highlighting, adding SEO details, or larger changes discussed with the author, the editor can make the edits directly in the author's branch and push the changes to `draft` branch, then rebuild the [draft site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io) to check them.
+   
+   To check out the author's branch locally: `git fetch origin` then `git checkout -b branch_name origin/branch_name`, which creates a new local branch that's linked to their remote branch. When you've made changes, push them back to `origin/branch_name`.
+
+2. Add tags to the blog post:
+
+   a. In the author's branch, update the [blogs_tags.json](./blog_tags.json) file by adding the slug of the blog post (not including the date part of the file name) to the start of the `posts` array (1-2 entries per line) for each appropriate tag. Do this in an editor (such as VSCode) and make sure the syntax is correct.
+
+   b. Push the changes to `draft` branch as before and check that they get built correctly on the [draft site](https://draft-openlibertyio.mybluemix.net/blog/).
+
+2. When a publishing date has been decided:
+
+   * Check that the post looks fine.
+   
+   * Check that the author's details and the SEO details, including front matter, the title, and the filename slug, are appropriate for the post.
+   
+   * Check that the post has tags defined in the `blogs_tags.json` file in the same PR.
+
+   * If necessary, rename the file with the planned publication date.
+
+3. On the day of publication (or the day before):
+
+   a. Approve the PR.
+   b. Ask @lauracowen (or another admin) to merge the PR into `staging`.
+   
+4. Request a build of the [staging OpenLiberty.io site from Travis CI](https://travis-ci.com/github/OpenLiberty/openliberty.io).
+
+5. When the build has finished, check to make sure the blog renders correctly on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/). 
+
+   This is the final check before the post is published live on the [production site](https://openliberty.io/blog/).
+
+   If there are any problems found on the [staging site](https://staging-openlibertyio.mybluemix.net/blog/), you must resolve them quickly or revert the PR.
+   
+   Make any changes in the author's branch, and push to both `draft` and `staging`.
+   
+6. To publish the post, create a PR from `staging` branch to `prod` branch and add @lauracowen (or other admin) as approver.
+
+7. When the PR is approved, merge it into `prod`.
+
+12. Rebuild the [production site from the IBM Cloud console](https://console.bluemix.net/devops/pipelines/fcc7c3e9-9c40-4a58-8a7f-09c08413ab7d?env_id=ibm:yp:us-south).
+
+    When the build has finished, check that the post looks as expected on [OpenLiberty.io/blog](https://openliberty.io/blog/).
+
+    If the post's file name uses a future date, the post will not display on the production site until that the production site is rebuilt on that day or later.
```

Comment:
"If the post's file name uses a future date, the post will not exist on the production site until at least that date and the production site has been rebuilt." or something like that